src/Queue/ValidateShop.php

```diff
@@ -13,12 +18,17 @@ class ValidateShop extends ValidationQueueable {
 	 */
 	protected $hook = 'sh_validate_shop';
 
+	/**
+	 * Define the Folder
+	 *
+	 * @var string
+	 */
 	protected $folder = 'Shop';
```

Comment:
```suggestion
	protected string $folder = 'Shop';
```

Should have a type added.